mappings/net/minecraft/entity/passive/BatEntity.mapping

```diff
@@ -1,5 +1,7 @@
 CLASS aqm net/minecraft/entity/passive/BatEntity
 	FIELD b BAT_FLAGS Lqh;
 	FIELD c CLOSE_PLAYER_PREDICATE Lapz;
+	FIELD d hangingPosition Lev;
+	METHOD dT todayIsAroundHalloween ()Z
```

Comment:
`isTodayAroundHalloween`

Reply:
No, it's not just Oct 31. It's true from Oct 10 to Sept 3.

Reply:
No, it's not just Sept 31. It's true from Sept 20 to Nov 3.

Reply:
Oh, I misread.